File: API/AcQua TaQuot/Shared/DataTransfertObject/Facturation/LigneFacturableKey.cs
namespace Shared.DataTransfertObject.Facturation
{
    public class LigneFacturableKey
    {
        public string Tache { get; set; }
        public Guid ModuleId { get; set; }
        public Guid TypeTacheId { get; set; }
        public Guid StatutId { get; set; }
    }
}

File: APP/TaQuot/src/models/Facturation/LigneFacturableKey.ts
import type LigneFacturableDTO from "./LigneFacturable";

export default class LigneFacturableKey {
  tache = '';
  moduleId = '';
  typeTacheId = '';
  statutId = '';

  public constructor(ligneFacturable : LigneFacturableDTO | null){
    if(ligneFacturable !== null)
    {
      this.tache = ligneFacturable.tache;
      this.moduleId = ligneFacturable.moduleId;
      this.typeTacheId = ligneFacturable.typeTacheId;
      this.statutId = ligneFacturable.statutId;
    }
  }
}